src/main/webapp/js/pages/index.js

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import Auth from '../lib/login.js';
```

Comment:
Did a linter re-sort this? If not, I believe that typically sort order is by file name, not by the import name (since you can import lots of things out of it).